### PR TITLE
Add a Gitter chat badge to README.md

### PR DESCRIPTION
**Pull request by gitter-badger**
### nodejs/automation now has a Chat Room on Gitter

@joyeecheung has just created a chat room. You can visit it here: [https://gitter.im/nodejs/automation](https://gitter.im/nodejs/automation?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&content=body_link).

This pull-request adds this badge to your README.md:

[Gitter](https://gitter.im/nodejs/automation?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge)

If my aim is a little off, please [let me know](https://github.com/gitterHQ/readme-badger/issues).

Happy chatting.

PS: [Click here](https://gitter.im/settings/badger/opt-out) if you would prefer not to receive automatic pull-requests from Gitter in future.

### CODE REVIEW

**Comment by joyeecheung** [comment]
@nodejs/automation Hi everyone, I have created [a gitter chat room](https://gitter.im/nodejs/automation) for our team, please LGTM or discuss here about whether you would like to use it for communication. If there are no strong objections (I'd say in a week?) then we can merge this.

**Comment by targos** [comment]
SGTM

**Comment by joyeecheung** [comment]
I have set up a hook that's supposed to send notifications to the room when there are activities in the repo. (https://github.com/nodejs/automation/settings/hooks/)

**Comment by maclover7** [comment]
+/- on IRC? I don't really have a strong opinion personally, just bringing up because that's the other main chat tool that's used

**Comment by joyeecheung** [comment]
@maclover7 To avoid losing notifications on IRC one has to set up a bounce server, that's a bit more hassle..

**Comment by Tiriel** [comment]
LGTM

**Review by Tiriel — APPROVED** [review]

**Comment by refack** [comment]
> @maclover7 To avoid losing notifications on IRC one has to set up a bounce server, that's a bit more hassle..

There is https://irccloud.com, but I don't know how the GFW likes it (BTW: the mobile app is free and essentially give you all the important paid features)

AFAIK The reason core is not using Gitter is "transparency" which is easier to achieve with IRC: http://logs.libuv.org/node-dev/latest. I'm not sure it's high-priority for this team.

**Comment by refack** [comment]
Which reminds me to add https://github.com/piscisaureus/slurp or #1

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Node.js Automation Team
 
+[![Join the chat at https://gitter.im/nodejs/automation](https://badges.gitter.im/nodejs/automation.svg)](https://gitter.im/nodejs/automation?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (refack) [inline README.md]: Can you remove the google analytics parameter (everything after the ?)

Comment (tniessen) [reply README.md]: Or including the `?` :wink:

**Review by tniessen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (joyeecheung) [reply README.md]: @refack @gitter-badger is a bot XD..I can take care of that while landing.

**Review by hiroppy — APPROVED** [review]

**Comment by joyeecheung** [comment]
A week has passed, nobody has objected, so I will merge this now. Thanks everyone!

**Pull request merged**